### ISSUE TITLE
[Suggestion] on select a pokemon show % capture probability

### ISSUE THREAD

**Issue by Xelwon** [kiideveloper/pokiimap#68]
It would be a good option to decide if go or not go to capture it :D.

Another good information to show would be CP, HP and IVs (AT, DEF, STA) 

An for human mode may be does not put it better.

**Comment by thebigsmileXD** [comment]
Sadly you can't calculate IV before you caught them, it depends on stardust and similar

**Comment by cris11368** [comment]
There's an xposed module that does this when you initiate the encounter. The information is there before you catch a pokemon, just doubt it's something that exist before you actually initiate the encounter though.

**Comment by Xelwon** [comment]
Exactly, the actual information stored for pokemons is level, base stats and IVs. 
Stardusts, candies, HP, CP  and level arc only are a represation on screen of this information.

**Comment by thebigsmileXD** [comment]
Thats interesting, so you could even in theory hide all pokemon with an iv less than 50%?
